tools/ovc/unit_tests/ovc/utils/args_to_string_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2018-2023 Intel Corporation
```

Comment:
may we should need to remove them from MO unit-test scope?

Reply:
Done.